src/composables/useEmailSubscription.ts

```diff
@@ -0,0 +1,120 @@
+import { getInstance } from '@snapshot-labs/lock/plugins/vue3';
+
+export function useEmailSubscription() {
+  enum Status {
+    wating,
+    success,
+    error
+  }
+
+  enum Level {
+    info = 'info',
+    warning = 'warning',
+    'warning-red' = 'warning-red'
+  }
+
+  const domain = {
+    name: 'snapshot',
+    version: '0.1.4'
+  };
+
+  const status: Ref<Status> = ref(Status.wating);
+  const postSubscribeLevel: Ref<Level> = ref(Level.info);
+  const postSubscribeMessage = ref('');
+  const loading = ref(false);
+  const auth = getInstance();
+  const { t } = useI18n();
+  const { web3 } = useWeb3();
+
+  async function sign(
+    message: {
+      email: string;
+      from: string;
+      timestamp: number;
+    },
+    types: any
+  ) {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
Why is this needed, I don't see any errors if I remove it

Reply:
the use of @ts-ignore raises an error